samples/drivers/lora/send/README.rst

```diff
@@ -0,0 +1,36 @@
+.. zephyr:code-sample:: lora-send
+   :name: LoRa send
+   :relevant-api: lora_api
+
+   Transmit a preconfigured payload every second using the LoRa radio.
+
+Overview
+********
+
+This sample demonstrates how to use the LoRa radio driver to configure
+the encoding settings and send data over the radio.
+
+Transmitted messages can be received by building and flashing the accompanying
+LoRa receive sample on another board.
```

Comment:
I think you didn't want to include the full Lora send commit in this PR. But this being said, you probably want to update the file anyway to cross-link to the receive sample.

Reply:
I made an error here, I already PR'd on this fork so didn't think it would be included

Reply:
Ya, still not looking right :) you probably want to rebase on top of main in interactive mode, drop the "send" commit altogether, and then do it again with just the change for the code-sample reference.

Reply:
Apologies - I'll sort it out!